parser/src/main/scala/aqua/parser/lift/Span.scala

```diff
@@ -51,11 +51,14 @@ object Span {
     str: String,
     idx: Int,
     len: Int
-  ): FocusedLine = FocusedLine(
-    str.substring(0, idx),
-    str.substring(idx, idx + len),
-    str.substring(idx + len)
-  )
+  ): FocusedLine =
+    val end = idx + len
```

Comment:
nit: Could you wrap the method in `{}`?
I don't like indentation as scope marking

Reply:
me too